core/src/main/java/com/sequenceiq/cloudbreak/core/bootstrap/service/host/ClusterHostServiceRunner.java

```diff
@@ -493,15 +494,19 @@ public void decoratePillarWithClouderaManagerCsds(Cluster cluster, Map<String, S
                         singletonMap("csd-urls", csdUrls))));
     }
 
-    public void decoratePillarWithClouderaManagerSettings(Map<String, SaltPillarProperties> servicePillar, ClouderaManagerRepo clouderaManagerRepo) {
+    public void decoratePillarWithClouderaManagerSettings(Map<String, SaltPillarProperties> servicePillar, ClouderaManagerRepo clouderaManagerRepo,
+            Stack stack) {
+        ServiceLocationMap serviceLocations = clusterApiConnectors.getConnector(stack.getCluster()).getServiceLocations();
         String cmVersion = clouderaManagerRepo.getVersion();
         servicePillar.put("cloudera-manager-settings", new SaltPillarProperties("/cloudera-manager/settings.sls",
-                singletonMap("cloudera-manager", singletonMap("settings", Map.of(
-                        "heartbeat_interval", cmHeartbeatInterval,
-                        "missed_heartbeat_interval", cmMissedHeartbeatInterval,
-                        "set_cdp_env", isVersionNewerOrEqualThanLimited(cmVersion, CLOUDERAMANAGER_VERSION_7_0_2),
-                        "deterministic_uid_gid", isVersionNewerOrEqualThanLimited(cmVersion, CLOUDERAMANAGER_VERSION_7_2_1),
-                        "cloudera_scm_sudo_access", CMRepositoryVersionUtil.isSudoAccessNeededForHostCertRotation(clouderaManagerRepo))))));
+                singletonMap("cloudera-manager", Map.of(
+                        "settings", Map.of(
+                                "heartbeat_interval", cmHeartbeatInterval,
+                                "missed_heartbeat_interval", cmMissedHeartbeatInterval,
+                                "set_cdp_env", isVersionNewerOrEqualThanLimited(cmVersion, CLOUDERAMANAGER_VERSION_7_0_2),
+                                "deterministic_uid_gid", isVersionNewerOrEqualThanLimited(cmVersion, CLOUDERAMANAGER_VERSION_7_2_1),
+                                "cloudera_scm_sudo_access", CMRepositoryVersionUtil.isSudoAccessNeededForHostCertRotation(clouderaManagerRepo)),
+                        "directories", serviceLocations.getAllVolumePath()))));
```

Comment:
Maybe we could call it mgmt_service_directories instead?

Reply:
sure, I will rename it